lib/plugins/aws/rollback.js

```diff
@@ -16,8 +16,8 @@ class AwsRollback {
     Object.assign(this, validate, setBucketName, updateStack, monitorStack);
 
     this.hooks = {
-      'before:rollback:initialize': () => BbPromise.bind(this).then(this.validate),
-      'rollback:rollback': () => {
+      'before:rollback:initialize': async () => BbPromise.bind(this).then(this.validate),
```

Comment:
I wonder, what's the point of putting `async/await` without refactoring promise chains?

Reply:
To reduce the scope - it's already quite a lot of functions to be refactored and these types of refactoring are error-prone. During the next step, we will be getting rid of promise chains as well, please see the description of the issue: https://github.com/serverless/serverless/issues/8368 and corresponding ones: https://github.com/serverless/serverless/issues/8369 and https://github.com/serverless/serverless/issues/7171

Reply:
@vladgolubev @pgrzesik just to reinforce the "error-prone" point, such a simple change (the one in request function above) broke one of the plugins we're using, see https://github.com/arabold/serverless-export-env/issues/26. So yeah, it's good to make it in steps.

Reply:
Ah, right @coyoteecd, that's an unfortunate side-effect of these migrations, but hopefully it's not going to impact too many plugins. I''ll bring up the topic so we can make sure to better announce such refactors/potential breaking changes to plugins in the future